Components/OracleMessage.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import LottieView from 'lottie-react-native';
+import { Platform } from 'react-native';
+
+import { Container, OracleText, MagicImage, ButtonText, RestartButton } from '../StyledComponents/OracleMessageStyling';
```

Comment:
No need to name your folder with capital letter, this convention is saved only for :

1. Names of components
2. Names of files of components